packages/docs/operations-manual/key-management.md

```diff
@@ -4,65 +4,209 @@
-| Vote signer key        | This key can be used to vote in Validator elections and for on-chain governance proposals.                                                                                                                                                                                          |
-| Attestation signer key | This key is used to sign attestations in Celo's lightweight identity protocol.                                                                                                                                                                                        |
+## Philosophy
+
+The Celo protocol was designed with the understanding that there is often an inherent tradeoff between the convenience of accessessing a private key and the security with which that private key can be custodied. As such, the protocol allows users to authorize private keys with specific, limited privileges. This allows users to custody each private key according to its sensitivity (i.e. what is the impact of this key being lost or stolen?) and its usage patterns (i.e. how often and under which circumstances will this key need to be accessed).
+
+
+## Summary
+
+The table below outlines a summary of the various key types in the Celo protocol. Note that these key types are *mutually exclusive*. A key that has been designated as one key type can never be used for a different purpose. For more details on a specific key type, please see that key type's specific section below.
+
+
+| Type                                | Description                                                                       | Ledger compatible |
+| ----------------------------------- |-----------------------------------------------------------------------------------|-------------------|
+| Undesignate                         | Default key type, cannot participate in proof-of-stake                            | Yes               |
+| Locked Gold Account (LGA)           | Used to lock and unlock cGLD and authorize other keys                             | Yes               |
+| Authorized vote signer              | Can vote on behalf of an LGA                                                      | Yes               |
+| Authorized validator (group) signer | Can register and manage a validator group on behalf of an LGA                     | No                |
+| Authorized validator signer         | Can register, manage a validator, and sign consensus messages on behalf of an LGA | No                |
+| Authorized validator BLS signer     | Used to sign blocks as a validator                                                | No                |
+| Authorized attestations signer      | Can sign attestations messages on behalf of a Locked Gold account                 | No                |
```

Comment:
I think its interesting to note that this technically is compatible, of course it would require one to touch the ledger with the spender app which I assume at some point there will be an option to do this?

Reply:
Yeah you can compile a "NO CONSENT" version and sideload that but it's not really an easy experience. If we want to have an attestations specific app that doesn't require confirmation the message prefix should probably be more attestations specific than it is already, at least.